2024-02-22 09:11:34,071 - __main__ - INFO - Message from bii: asdasd
2024-02-22 09:12:12,535 - TeleBot - ERROR - Infinity polling: polling exited
2024-02-22 09:12:12,536 - TeleBot - ERROR - Break infinity polling
2024-02-22 09:12:15,802 - __main__ - INFO - Message from bii: sad
2024-02-22 09:14:08,451 - TeleBot - ERROR - Infinity polling: polling exited
2024-02-22 09:14:08,452 - TeleBot - ERROR - Break infinity polling
2024-02-22 09:14:12,719 - __main__ - INFO - Message from bii: ds
2024-02-22 09:14:16,623 - TeleBot - ERROR - Infinity polling exception: runStatistics() takes 0 positional arguments but 1 was given
2024-02-22 09:14:16,624 - TeleBot - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/__init__.py", line 1016, in infinity_polling
    self.polling(non_stop=True, timeout=timeout, long_polling_timeout=long_polling_timeout,
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/__init__.py", line 1104, in polling
    self.__threaded_polling(non_stop=non_stop, interval=interval, timeout=timeout, long_polling_timeout=long_polling_timeout,
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/__init__.py", line 1179, in __threaded_polling
    raise e
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/__init__.py", line 1141, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/util.py", line 149, in raise_exceptions
    raise self.exception_info
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/util.py", line 92, in run
    task(*args, **kwargs)
  File "/home/biibon/python/venv/lib/python3.10/site-packages/telebot/__init__.py", line 7868, in _run_middlewares_and_handler
    result = handler['function'](message)
TypeError: runStatistics() takes 0 positional arguments but 1 was given

2024-02-22 09:14:43,836 - TeleBot - ERROR - Infinity polling: polling exited
2024-02-22 09:14:43,837 - TeleBot - ERROR - Break infinity polling
2024-02-22 09:14:57,763 - __main__ - INFO - Message from bii: asdhasdh
2024-02-22 09:15:00,673 - __main__ - INFO - Message from bii: sdasd
2024-02-22 09:15:02,468 - TeleBot - ERROR - Infinity polling: polling exited
2024-02-22 09:15:02,468 - TeleBot - ERROR - Break infinity polling
2024-02-22 09:15:05,974 - __main__ - INFO - Message from bii: asdasd
